app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt

```diff
@@ -215,13 +215,13 @@ class AddProfileActivityTest {
       )
       onView(withId(R.id.add_profile_activity_create_button)).perform(scrollTo())
       onView(withId(R.id.add_profile_activity_create_button)).perform(click())
-      intended(hasComponent(ProfileActivity::class.java.name))
+      intended(hasComponent(ProfileChooserActivity::class.java.name))
     }
   }
 
   /* ktlint-disable max-line-length */
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputName_inputPin_clickCreate_checkOpensProfileActivity() {
+  fun testProfileChooserActivity_changeConfiguration_inputName_inputPin_clickCreate_checkOpensProfileActivity() {
```

Comment:
Rename to `testAddProfileActivity_changeConfiguration_inputName_inputPin_clickCreate_checkOpensProfileChooserActivity `